cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -256,12 +277,21 @@ def test_vmware_inaccessible_datastore_vm_provisioning():
             2.See all available templates
             3.CFME should provision VM on datastore other than the one that is inaccessible.
     """
-    pass
+    inaccessible_datastores = [
+        datastore for datastore in provider.mgmt.list_datastore()
+        if not provider.mgmt.get_datastore(datastore).summary.accessible]
```

Comment:
**Question** How often are data stores inaccessible on our providers? Would it be better to have a dummy datastore that is mark as inaccessible?

Reply:
Each of our VMware envs do have least one inaccessible datastores and sometimes in rare cases we may have more.